packages/grafana-ui/src/components/FormField/FormField.tsx

```diff
@@ -13,7 +13,7 @@ export interface Props extends InputHTMLAttributes<HTMLInputElement> {
 
 const defaultProps = {
   labelWidth: 6,
-  inputWidth: 12,
+  // inputWidth: 12,
```

Comment:
hmm need to remove or put to separate PR maybe. Problem is that you cannot set FormField with `100%` width because of that. If you do not specify inputs width it will be `100%` width but this will always set something there. I think you can do `inputWidth={null}` to do it but seems just like a bad default value here.